Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.cpp

```diff
@@ -888,8 +888,8 @@ namespace AZ::AtomBridge
             const float stepAngle = DegToRad(angularStepDegrees);
             const float startAngle = DegToRad(startAngleDegrees);
             const float stopAngle = DegToRad(sweepAngleDegrees) + startAngle;
-            SingleColorDynamicSizeLineHelper lines(1+static_cast<int>(sweepAngleDegrees/angularStepDegrees));
             AZ::Vector3 radiusV3 = AZ::Vector3(radius);
+            SingleColorDynamicSizeLineHelper lines(1+static_cast<int>(AZStd::abs(sweepAngleDegrees)/angularStepDegrees));
```

Comment:
Maybe could do the abs in the constructor of SingleColorDynamicSizeLineHelper instead? Then it would catch any similar issue.

Reply:
+1